03/app.js

```diff
@@ -1,14 +1,55 @@
-console.log('DOM');
+console.log("DOM");
 
 const buttonSettings = {
-    attr: {
-        className: 'btn',
-        title: 'super button'
-    },
-    css: {
-        border: '1px solid #336699',
-        padding: '5px 20px',
-        color: '#444'
-    },
-    text: 'Click me!',
-}
\ No newline at end of file
+  attr: {
+    className: "btn",
+    title: "super button",
+  },
+  css: {
+    border: "1px solid #336699",
+    padding: "5px 20px",
+    color: "#444",
+  },
+  text: "Click me!",
+};
+
+const parent = document.querySelector(".parent-for-button");
+
+if (parent) {
+  const button = document.createElement("button");
+
+  for (key in buttonSettings) {
+    switch (key) {
+      case "attr":
+        const attributes = buttonSettings[key];
+
+        for (attr in attributes) {
+          const attrValue = attributes[attr];
+
+          switch (attr) {
+            case "className":
+              button.classList.add(attrValue);
+              break;
+            case "title":
+              button.setAttribute(attr, attrValue);
+              break;
+          }
```

Comment:
Nie dało by się prościej? `button[attr] = attrValue;`

Reply:
A czy to nie doda atrybutu className, zamiast class?